### DISCUSSION TITLE
Deploy on something

### DISCUSSION THREAD

**Discussion by vadim9999** [atherdon/dashb#102] [Ideas]
Deploy on AWS Amplify

**Comment by atherdon** [comment]
@vadim9999 can you give us more details about what we are looking here?

**Comment by atherdon** [comment]
I was thinking about AWS amplify. to be honest - I don't like it.
I mean - it sounds pretty awesome, but for me - it's just yet another framework that you spend time to master it, and then it'll be gone in few years.

I think we'll benefit more from was lambda/serverless approach.
let me explore it a bit more. but it sounds more exciting for me

**Comment by atherdon** [comment]
I have another idea. While, probably I still want to stick with DIY approach, [Webiny](https://www.webiny.com/serverless-app/headless-cms/) purchased our thematic newsletters.

I still don't wanna to be tight up with something that I don't know how effective will be - but I'll try at least part of their stuff and then will think about it.

**Reply by atherdon** [reply]
We're not ready for using Webiny as well
